rpc/src/test/scala/org/bitcoins/rpc/TestUtil.scala

```diff
@@ -73,7 +73,11 @@ trait TestUtil extends BitcoinSLogger {
     } else firstAttempt
   }
 
-  def startServers(servers: Vector[BitcoindRpcClient])(implicit system: ActorSystem): Unit = {
+  /**
+   * Starts the given servers, and waits for them (in a blocking manner)
+   * until they are started.
+   */
+  def startServers(servers: BitcoindRpcClient*)(implicit system: ActorSystem): Unit = {
```

Comment:
Allright. Var args are cool so I wanted to use them here, didn't know they had performance implications!

Reply:
It should be noted that they *can* -- but not necessarily -- have them. A bigger goal of mine is to improve performance in bitcoin-s, which means limiting what is possible in certain areas of the codebase. 

Somehthing like varargs if hard to reason about because you can pass in a lot of different things! :-). This is just test code though, so it isn't a huge concern here.

Reply:
That totally makes sense!